### PR TITLE
Attribution

### PR DESCRIPTION
**Pull request by dpatchigolla**
## Description of the change

Added Multi-touch attribution model v1. 

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: src/profiles-pycorelib/attribution_model.py [hunk 1]
```diff
@@ -0,0 +1,239 @@
+# Commenting out this model as this has some breaking changes for v0.10.4
```
Comment (shubhammehra4) [inline src/profiles-pycorelib/attribution_model.py:1]: Nit Pick: Let's get rid of this comment

File: src/profiles-pycorelib/attribution_model.py [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+    # json schema for the build spec
+    BuildSpecSchema = {
+        "type": "object",
+        "properties": {
+            "touchpoint_var" : { "type": "string" }, # model
+            "days_since_first_seen_var": { "type": "string" }, # model
+            "first_seen_since": { "type": "integer" },
+            "conversion_entity_var": { "type": "string" }, #model
+            "output_field": { "type": "string" },          
+            "entity": { "type": "string"}
```
Comment (shubhammehra4) [inline src/profiles-pycorelib/attribution_model.py:26]:
```suggestion
            **EntityKeyBuildSpecSchema["properties"],
```
Suggested to use the schemas provided for standard wht properties

**Review by shubhammehra4 — APPROVED** [review]
LGTM. Some minor comments

File: src/profiles-pycorelib/attribution_model.py (hunk 2 above)
Comment (shubhammehra4) [reply src/profiles-pycorelib/attribution_model.py:26]: Can ignore if this isn't used for declaring the entity key, might've misunderstood it

Comment (dpatchigolla) [reply src/profiles-pycorelib/attribution_model.py:26]: Yes, it's not to declare the entity, but knowing what the entity is and use that to refer to the entity var model

**Pull request closed without merging**